input/pagecontent/developerGuide.md

```diff
@@ -4,9 +4,9 @@
 
 In order to make requests against this API clients must:
 
-- Obtain an access token from the OAuth token endpoint using the client credentials issued to their organisation,  
+- Obtain an access token from the OAuth token endpoint using the client credentials issued to their organisation,
 
-- Pass the access token as a bearer token in the `Authorization` header of all requests, along with an API key in the `X-Api-Key` header,  
+- Pass the access token as a bearer token in the `Authorization` header of all requests, along with a `X-Correlation-Id` header,
```

Comment:
The existing API requires the API key, perhaps this document can reflect both scenarios?

Reply:
The current OAS is not used by the NIA/HIP ingress, so I am engineering this for Digital Services Hub (DSH) and connector plane only. If I include the Api-Key it will not be accepted by the provider onboarding. i.e. the IG, being publicly accessible is for Sector Application consumers, any internal access would need to be documented separately?

Reply:
> the IG, being publicly accessible is for Sector Application consumers

Existing sector consumers use the existing API, and are required to provide an api key though - they refer to documentation in this IG (not internal access). It is not a big issue to hold this up though